src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -579,4 +555,8 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     return false;
   }
 
+  private HostAndPort generateHostAndPort(List<Object> hostInfos) {
```

Comment:
@HeartSaVioR weird?. Even though the PR shows that the method has been added, I only see it once here: https://github.com/charsyam/jedis/blob/feature/using_cluster_slots/src/main/java/redis/clients/jedis/JedisClusterInfoCache.java